lib/fake-app.js

```diff
@@ -123,7 +148,7 @@ class FakeApp {
   }
 
   getScreenshot () {
-    return 'hahahanotreallyascreenshot';
+    return fs.readFileSync(SCREENSHOT, 'base64');
```

Comment:
cannot we generate the image in runtime using pngjs?

Reply:
we could but the main reason i am putting this here is to test image processing stuff against it so it needs to be a known image. i'll see if i can reduce the size.

Reply:
OK i reduced the size